bin/peggy.mjs

```diff
@@ -0,0 +1,331 @@
+import { Command, Option } from "commander";
+import fs from "fs";
+import path from "path";
+import { default as peg } from "../lib/peg.js";
+import util from "util";
+
+const MODULE_FORMATS = ["amd", "commonjs", "es", "globals", "umd"];
+const MODULE_FORMATS_WITH_DEPS = ["amd", "commonjs", "es", "umd"];
+const DEFAULT_FORMAT = "COMMON_JS";
+let verbose = false;
+
+// Helpers
+
+function abort(message) {
+  console.error(message);
+  process.exit(1);
+}
+
+function abortError(msg, error) {
+  console.error(msg);
+  if (typeof error.format === "function") {
+    abort(error.format([{
+      source: testGrammarSource,
+      text: testText,
+    }]));
+  } else {
+    if (verbose) {
+      abort(error);
+    } else {
+      abort(`Error: ${error.message}`);
+    }
+  }
+}
+
+function addExtraOptions(options, json) {
+  let extraOptions;
+
+  try {
+    extraOptions = JSON.parse(json);
+  } catch (e) {
+    if (!(e instanceof SyntaxError)) { throw e; }
+
+    abortError("Error parsing JSON:", e);
+  }
+  if ((extraOptions === null)
+      || (typeof extraOptions !== "object")
```

Comment:
Aren't extra brackets prohibited by the linting rules?
```suggestion
  if (extraOptions === null
      || typeof extraOptions !== "object"
```

Reply:
No.

https://github.com/peggyjs/peggyjs-eslint-config/blob/46d7fcef3477325971c9c79a5c49717d960bc1cd/index.js#L59

I'm open to changing it, but I often use lots of extra parens so that the reader doesn't have to know as much about operator precedence.

Reply:
Ok. I'm personally tend to not put extra braces around comparison operators, but that not a blocker